execution/setContext.sh

```diff
@@ -274,7 +280,18 @@ if [[ -n "${AZID}" ]]; then
         . ${AUTOMATION_DIR}/setCredentials.sh "${ACCOUNT}"
 
     fi
-    az account set --subscription "${AZID}" --output none || exit $?
+
+    az_cli_args=()
+    # -- Only show errors unless debugging --
+    if willLog "${LOG_LEVEL_DEBUG}"  ]]; then
+        az_cli_args+=("--output" "json" )
+        echo "debug output"
+    else
+        az_cli_args+=("--output" "none" )
+        echo "normal output"
```

Comment:
Do we want this trace every time?

Reply:
If we do display it the subscription details of the subscription will be shown any time that setContext is run... would get noisey

Reply:
Oh never mind no we don't I'll get rid of that